addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1188,7 +1188,9 @@ var SnippetEditor = Widget.extend({
             over: function () {
                 if (self.dropped) {
                     self.$target.detach();
-                    $('.oe_drop_zone').removeClass('invisible');
+                    for (const dropzoneEl of self.$body[0].querySelectorAll(".oe_drop_zone")) {
+                        dropzoneEl.classList.remove("invisible");
+                    }
```

Comment:
In fact we already have the dropzones in a variable so we didn't need to search for them again in the DOM at this point. Maybe we could do something like that:
```js
this.$dropZones.forEach(dropzoneEl => dropzoneEl.classList.remove("invisible"));
```

Reply:
@bvr-odoo Indeed! However, let's note that at this place in the code `this` is referring to the dropzone so we can not do `this.$dropZones`, however, we could do something like this :
``` diff
+        const dropZoneEls = [...this.$dropZones];
         this.$dropZones.droppable({
             over: function () {
                 if (self.dropped) {
                     self.$target.detach();
-                    $('.oe_drop_zone').removeClass('invisible');
+                    dropZoneEls.forEach(dropzoneEl => dropzoneEl.classList.remove("invisible"));
                 }
```

Reply:
as explained here https://github.com/odoo/odoo/pull/136644#discussion_r1341247767
there is no bug when we drag and drop the snippet thanks to its move button as the drag and drop starts inside the iframe